app/frontend/stores/root-store.ts

```diff
@@ -88,11 +92,6 @@ export const RootStoreModel = types
       self.userChannelConsumer?.consumer?.disconnect()
     },
   }))
-  .actions((self) => ({
-    afterCreate() {
-      self.loadLocalPersistedData()
-    },
-  }))
```

Comment:
removing this seems like it might have unintended consequences. e.g. for the `afterLoginPath` - looks like this needs to persist while the user is logged out?

Reply:
ah, I will leave this then.